src/PoolManager.sol

```diff
@@ -88,11 +88,6 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC6909Claims {
         return pools[id].positions.get(_owner, tickLower, tickUpper);
     }
 
-    /// @inheritdoc IPoolManager
-    function getLocker() external view override returns (address locker) {
```

Comment:
Instead of `getLocker` we should probably expose `isLocked` here so people can see if the contract is locked or not